.github/workflows/selftest.yml

```diff
@@ -23,11 +23,12 @@ jobs:
         uses: ./
         with:
           directory: test-artifacts
+          distro: stream8
```

Comment:
It would be nice if the name would be 'el8stream' throughout the PR - this would match the naming that we use in OST already.

Reply:
Done